src/arrayMethodJoin.js

```diff
@@ -4,8 +4,24 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let string = '';
+
+    for (let i = 0; i < this.length; i++) {
+      let element = '';
+
+      if (this[i] != null) {
+        element = this[i];
```

Comment:
You don't need to overwrite new variable `element` here. You can overwrite `string` directly